packages/gatsby-image/src/index.js

```diff
@@ -480,6 +480,8 @@ class Image extends React.Component {
           style={{
             position: `relative`,
             overflow: `hidden`,
+            maxWidth: image.maxWidth ? `${image.maxWidth}px` : null,
```

Comment:
@wardpeet testing it against my test site, I think the issue is here.  No need to use `${image.maxWidth}px` if you log maxWidth/maxHeight it already has px appended.  When I remove that and just use `image.maxWidth/image.maxHeight` it works again. Thought not sure how it is working for you? :)

Reply:
I also think that is why the PropType for `maxHeight/maxWidth` were both `string` instead of number under `fluidObject`.  As I'm not getting an error for invalid prop on those being number.